### PR TITLE
Vidit msft/regenerate sdk for sql

### PR DESCRIPTION
**Pull request by vidit-msft**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Review by archerzz — CHANGES_REQUESTED** [review]
@vidit-msft Please update the API: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/sqlmanagement/Azure.ResourceManager.Sql/api/Azure.ResourceManager.Sql.netstandard2.0.cs

**Comment by vidit-msft** [comment]
Thanks @archerzz , I have updated the API,

**Review by archerzz — APPROVED** [review]

File: sdk/sqlmanagement/Azure.ResourceManager.Sql/tests/Scenario/SqlDatabaseTests.cs [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.Resources;
+using Azure.ResourceManager.Resources.Models;
+using Azure.ResourceManager.Sql.Models;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.Sql.Tests.Scenario
+{
+    public class SqlDatabaseTests : SqlManagementClientBase
```
Comment (Yao725) [inline sdk/sqlmanagement/Azure.ResourceManager.Sql/tests/Scenario/SqlDatabaseTests.cs:16]: The CI failed due to the test failure, please re-record the tests here. Two things to note are:

- Please record all tests at once. `SqlDatabaseApiTests` and `SqlDatabaseApiTestsWithEnclaves` share the common `GlobalSetUp`. Record the tests one by one will cause the record corresponding to GlobalSetUp to be rewritten and all previous tests will fail in playback mode.
- Please also record the tests both with [sync and async methods](https://github.com/Azure/azure-sdk-for-net/blame/main/sdk/resourcemanager/Azure.ResourceManager/docs/TestGuide.md#L59), there is no sync method test recording.

Comment (vidit-msft) [reply sdk/sqlmanagement/Azure.ResourceManager.Sql/tests/Scenario/SqlDatabaseTests.cs:16]: Thanks @Yao725 , I have re-recorded the tests and added both the sync and async methods.

**Review by Yao725 — APPROVED** [review]

**Pull request merged**